docs/workflows/quick-start-guide.md

```diff
@@ -1,52 +1,131 @@
 
 # Quickstart Guide
 
+## Dependencies
+
+Please ensure you have the following dependencies installed:
+
+* [gen3-client](../requirements/gen3-client.md)
+* [gen3-tracker](../requirements/gen3-tracker.md)
+
+## Usage
+
+`g3t  [OPTIONS] COMMAND [ARGS]...`
+
+The following options and environmental variables are synonymous, you may set them as environmental variables or pass them as parameters to the command line.
+
+| option       | environment     | comment             | example       |
+|--------------|-----------------| ------------------- |---------------|
+| --project_id | G3T_PROJECT_ID  | authorization       | aced-test     |
+| --profile    | G3T_PROFILE     | gen3-client profile | production    |
+| --format     | G3T_FORMAT      | Output format.      | default: yaml |
+
+Alternatively, you can set the environmental variables using the `EXPORT` function e.g.:
+
+```sh
+export G3T_PROJECT_ID=aced-myproject
+```
+
+
 ## 1. Upload Data (To an Existing Project on ACED IDP)
-This section outlines how to add data to ACED IDP. Many `g3t` commands behave similarily to `git` commands
+This section outlines how to add data to ACED IDP. Many `g3t` commands behave similarly to `git` commands
+
+### Initialize a new project:
 
-Initialize a new project
 ```
 g3t init aced-demo
```

Comment:
even after setting `G3T_PROFILE=local`, I get
```
WARNING: No --profile specified, found multiple gen3_client profiles: ['local', 'dev', 'staging', 'cbds-prod', 'cbds-dev']
No profile set. Continuing in disconnected mode. Use `set profile <profile>`
Initialized empty Git repository in /Users/wongq/data/cbds-demo/local/cbds-for_docs/.git/
```

should we make it `g3t --profile <profile_name> init aced-demo`?